slave.mk

```diff
@@ -203,7 +203,7 @@ $(addprefix $(DEBS_PATH)/, $(SONIC_DPKG_DEBS)) : $(DEBS_PATH)/% : .platform $$(a
 	# Build project
 	pushd $($*_SRC_PATH) $(LOG)
 	[ ! -f ./autogen.sh ] || ./autogen.sh $(LOG)
-	dpkg-buildpackage -rfakeroot -b -us -uc -j$(SONIC_CONFIG_MAKE_JOBS) $(LOG)
+	dpkg-buildpackage -rfakeroot -b -us -uc -j$(SONIC_CONFIG_MAKE_JOBS) --as-root -T$($*_DPKG_TARGET) $(LOG)
```

Comment:
>--as-root -T [](start = 69, length = 12)

I could find the usage for the 2 parameters. I am not clear the problem trying to solve. Could you clarify?

Reply:
man dpkg-buildpackage

```
 3. If  a  specific  target has been selected with the -T or --target option, it calls that target and stops here. Otherwise it runs the preclean hook and calls fakeroot debian/rules clean to clean the build-tree (unless -nc is special field).


       --as-root
              Only meaningful together with --target (since dpkg 1.15.0).  Requires that the target be run with root rights.
```

this is needed because we need to build a special target for syncd-vs.

ideally, we can seperate the vslib completely from sairedis repo, then we can first build vslib and the install it and then build syncd. but, it is not an option right now.

Reply:
Thanks!